script/lib/js-yaml.min.js

```diff
@@ -1,2 +0,0 @@
-/*! js-yaml 4.1.0 https://github.com/nodeca/js-yaml @license MIT */
```

Comment:
Do we need to keep lib directory if it's empty?

Reply:
It's not empty. It has [other files as well](https://github.com/padaiyal/sanitizer/tree/issue19/script/lib).